src/current/_includes/v24.2/misc/session-vars.md

```diff
@@ -47,13 +47,16 @@
 | <a id="multiple-active-portals-enabled"></a> `multiple_active_portals_enabled` | Whether to enable the [multiple active portals]({% link {{ page.version.version }}/postgresql-compatibility.md %}#multiple-active-portals) pgwire feature. | `false` | Yes | Yes |
 | <a id="node-id"></a> `node_id` | The ID of the node currently connected to. <br/><br/>This variable is particularly useful for verifying load balanced connections. | Node-dependent | No | Yes |
 | <a id="null-ordered-last"></a> `null_ordered_last` | Set the default ordering of `NULL`s. The default order is `NULL`s first for ascending order and `NULL`s last for descending order. | `false` | Yes | Yes |
-| <a id="optimizer-merge-joins-enabled"></a> `optimizer_merge_joins_enabled` | If `on`, the optimizer will explore query plans with merge joins. | `on` | Yes| Yes |
+| <a id="optimizer-merge-joins-enabled"></a> `optimizer_merge_joins_enabled` | If `on`, the optimizer will explore query plans with merge joins. | `on` | Yes | Yes |
+| <a id="optimizer-push-offset-into-index-join"></a> `optimizer_push_offset_into_index_join` | If `on`, the optimizer will attempt to push offset expressions into index [join expressions]({% link {{ page.version.version }}/joins.md %}) to produce more efficient query plans. | `on` | Yes | Yes |
```

Comment:
should the link be "index join expressions" instead of just "join expressions"?

Reply:
Actually, I'd probably remove the link altogether since the linked page doesn't talk about index joins at all

Reply:
Done.